TodoListManagerTest/src/test/java/PerformanceTest/TodosPerformanceTest.java

```diff
@@ -0,0 +1,170 @@
+package PerformanceTest;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.json.JSONObject;
+import org.junit.jupiter.api.*;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import com.sun.management.OperatingSystemMXBean;
+
+public class TodosPerformanceTest {
+    private static final String pUrl = "http://localhost:4567/todos";
+    private static ProcessBuilder pb;
+    private final int[] populationSize = {1,10,50,100,200,500,1000};
+    private OperatingSystemMXBean osBean = (OperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean();
+    private String newTodoId = "1";
+
+    @BeforeAll
+    static void setupProcess() {
+        String os = System.getProperty("os.name");
+        if (os.toLowerCase().contains("windows")) {
+            pb = new ProcessBuilder(
+                    "cmd.exe", "/c", "java -jar .\\src\\test\\resources\\runTodoManagerRestAPI-1.5.5.jar");
+        }
+        else {
+            pb = new ProcessBuilder(
+                    "sh", "-c", "java -jar ./src/test/resources/runTodoManagerRestAPI-1.5.5.jar");
+        }
+    }
+
+    @BeforeEach
+    void startServer() throws InterruptedException {
+        try {
+            pb.start();
+            Thread.sleep(1000);
+        } catch (IOException e) {
+            System.out.println("No server");
+        }
+    }
+
+    @AfterEach
+    void shutServer() {
+        try {
+            RestAssured.get("http://localhost:4567/shutdown");
+        }
+        catch (Exception ignored) {
+        }
+    }
+
+    @Test
+    public void CreateTodosJSONPost(){
+        int max = populationSize[populationSize.length - 1];
+        double[] time = new double[populationSize.length];
+        double[] cpuUsage = new double[populationSize.length];
+        long[] freeMemory = new long[populationSize.length];
+        int targetIndex = 0;
+
+        for(int i=1; i<=max;i++){
+            JSONObject body = new JSONObject();
+            String newTitle = "New Todo " + i;
+            String newDescription = "New Description";
+            boolean newDoneStatus = false;
+            body.put("title",newTitle);
+            body.put("doneStatus",newDoneStatus);
```

Comment:
if newDoneStatus is always false why not just put false as parameter instead of creating a new variable, just my proposition.